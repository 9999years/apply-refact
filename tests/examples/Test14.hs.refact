[("tests/examples/Test14.hs:1:1: Warning: Use camelCase\nFound:\n  type Ann_Test = Int\nWhy not:\n  type AnnTest = Int\n",[])]
