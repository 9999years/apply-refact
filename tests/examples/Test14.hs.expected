type Ann_Test = Int
